### DISCUSSION TITLE
[Feature]Managed Identity support for local executions

### DISCUSSION THREAD

**Discussion by bjowol** [microsoft/vscode-azureAutomation#44] [Ideas]
Would like for the scripts running locally, either in debug or not, to support managed identity authentication.

**Comment by mortenlerudjordet** [comment]
MSI uses a protected endpoint only available inside azure, so can't be used locally.
Locally would need to be a VM inside Azure or some other device residing inside a vNet.

One solution would be to use code like suggested here:
https://github.com/microsoft/vscode-azureAutomation/discussions/46
